latest/docker-initialize.py

```diff
@@ -227,23 +232,27 @@ def setup(self, **kwargs):
 BUILDOUT_TEMPLATE = """
 [buildout]
 extends = buildout.cfg
+user=admin:{password}
```

Comment:
this overrides https://github.com/senaite/senaite.docker/blob/353c42805c683b2c7c3ab7dcf046689e15f20d39/latest/buildout.cfg#L12